src/portal/src/app/project/project-detail/project-detail.component.ts

```diff
@@ -131,6 +136,32 @@ export class ProjectDetailComponent implements OnInit {
   ngOnInit() {
     this.projectId = this.route.snapshot.params['id'];
     this.getPermissionsList(this.projectId);
+    if (!this._subscription) {
+      this._subscription = this._subject.pipe(
+          debounceTime(100)
+      ).subscribe(
+          type => {
+            if (type === DOWN) {
+              this.resetTabsForDownSize();
+            } else {
+              this.resetTabsForUpSize();
+            }
+          }
+      );
+    }
+  }
+
+  ngAfterViewInit() {
+    this.previousWindowWidth = window.innerWidth;
+    setTimeout(() => {
+      this.resetTabsForDownSize();
+    }, 10);
```

Comment:
here 10 is a random number, I think set it to 0 will also work.

Reply:
> here 10 is a random number, I think set it to 0 will also work.
Agreed!!!